pkg/chart/loader/local.go

```diff
@@ -0,0 +1,75 @@
+/*
+Copyright The Helm Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package loader
+
+import (
+	"errors"
+	"os"
+	"path/filepath"
+	"strings"
+)
+
+// ExpandFilePath expands a local file, dir or glob path to a list of files
+func ExpandFilePath(path string) ([]string, error) {
```

Comment:
`action` does seem like a more suitable place :)
I don't remember what was the reason it was placed in the `chart/loader` package, but I `ExpandFilePath` was public only because it's used in the `action` package.
It indeed seems like it doesn't need to be public.